cubic-server/protocol/ClientPackets.cpp

```diff
@@ -187,12 +161,7 @@ std::shared_ptr<std::vector<uint8_t>> protocol::createChunkDataAndLightUpdate(co
 std::shared_ptr<std::vector<uint8_t>> protocol::createWorldEvent(const WorldEvent &in)
 {
     std::vector<uint8_t> payload;
-    serialize(payload,
-        in.event, addInt,
-        in.position, addPosition,
-        in.data, addInt,
-        in.disableRelativeVolume, addBoolean
-    );
+    serialize(payload, in.event, addInt, in.position, addPosition, in.data, addInt, in.disableRelativeVolume, addBoolean);
```

Comment:
oof²

Reply:
Some code would need to be formatted by hand and then deactivate the clang-format on this temporally.